src/psyclone/dynamo0p3.py

```diff
@@ -7533,6 +7595,25 @@ def _setup(self, ktype, module_name, args, parent):
         # Properties of the mesh required by this kernel
         self._mesh_properties = ktype.mesh
 
+    def _initialise_argument_kinds(self):
```

Comment:
This is only called once and, since it consists of only three lines, please could you inline it into the constructor.

Reply:
Done.